prime-router/docs/observability/azure-events.md

```diff
@@ -60,3 +96,87 @@ customEvents
 | where name == "MyEvent"
 | where customDimensions.importantString == "important"
 ```
+
+## Common Queries in KQL
```

Comment:
Maybe add an example of how to get the number of observations containing any flu code?

Reply:
Like we discussed yesterday? Sure